### ISSUE TITLE
[BUG] User who created the contest should be able to view its submissions

### ISSUE THREAD

**Issue by dgarvit** [omegaup/omegaup#1843]
### Current behaviour
User who creates a contest himself can't access the submissions of the contest (gives an error 404 on page but the console says 403 forbidden)

### Expected behaviour
User who creates the contest should be able to view its submissions

### Steps to reproduce
* Create a new contest
* Go to submissions

**Comment by Sugirjan** [comment]
I hope we can get the submissions. Did you try this?
![submissions](https://user-images.githubusercontent.com/17314165/37446022-02179d50-2841-11e8-8455-9e15974721e0.PNG)

Also, you can add '/admin' to the URL like 'https://omegaup.com/arena/ContestName/admin'

**Comment by dgarvit** [comment]
Yes I tried this. The page shows 404.
The console says error 403 FORBIDDEN.

**Comment by dgarvit** [comment]
This error happens when there is no submission so far.
I think rather than giving such an error even to the administrator, it would be better to show that no submissions have been made so far.

**Comment by Sugirjan** [comment]
First, you have to open the contest like a regular contestant. After that, you can access submissions as an admin.

**Comment by dgarvit** [comment]
Yes, I did exactly that, and got that error. Both on my local server and on
the omegaUp website.

**Comment by lhchavez** [comment]
on the VM if i log in as the admin user and go to http://localhost:8080/arena/prueba and register and then navigate to http://localhost:8080/arena/prueba/admin#runs , i am able to see the runs from all the users. is that not the case?

you might need to run `./stuff/bootstrap-environment.py --purge` to populate that contest with some runs, though.

in any case, i agree that having to visit the normal contest _as a contestant_ while you're an admin is kinda broken :( we haven't figured out a way to satisfy all the constraints, because some admins also want to participate in their own contests and would like to appear in the ranking.

**Comment by dgarvit** [comment]
Yes, that is required. Wouldn't it be nice if we rather send something softer than 400 error? I'd suggest a page which says 0 submissions so far would be better. Your thoughts?

**Comment by dgarvit** [comment]
For visiting the page as a contestant while being an admin. I can suggest one solution. We can ask the contest owner while making the contest if he wants to participate or not, and this add the necessary functioning.
In some platforms (like some of the native ones we have built at my institute), the problem setter/owner of the contest is not allowed to give the contest. Though I agree it is a nice feature for allowing them to participate too.

**Comment by lhchavez** [comment]
huh, i am unable to reproduce the issue on the local VM O_O

to start we can redirect to the non-admin view instead of showing a 403. for the rest of the functionality ideally we probably want to let the admin submit runs that do not count against the scoreboard if they enter through `/admin`, but do if they enter as a normal contestant.

**Comment by dgarvit** [comment]
Yes, that sounds like a nice solution!
